pkgs/development/python-modules/grpcio-tools/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, fetchPypi, lib, grpc, grpcio}:
+
+buildPythonPackage rec {
+  pname = "grpcio-tools";
+  version = "1.13.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1ck6818kb4nb6skm9lqg492brqs7kfk65f4hh2c7h7c8pkbrpcw1";
+  };
+
+  enableParallelBuilding = true;
+
+  propagatedBuildInputs = [ grpc grpcio ];
+
+  doCheck = false;
```

Comment:
Why do you disable tests? Please provide a comment with the reason.

Reply:
done